src/reducers/entities/posts.js

```diff
@@ -103,6 +127,11 @@ function handlePendingPosts(pendingPostIds = [], action) {
         }
         return nextPendingPostIds;
     }
+    case PostTypes.REMOVE_PENDING_POST: {
+        const postId = action.data.id;
+        const nextPendingPostIds = pendingPostIds.filter((post) => post !== postId);
```

Comment:
Can we change `post` to `pendingPostId` just not to confuse post ID with typical post object?

Reply:
You are right :)

Reply:
Done :)

Reply:
I mean, can we have it something like:
```
const nextPendingPostIds = pendingPostIds.filter((pendingPostId) => pendingPostId !== postId);
```

Reply:
Ohh, sorry my misunderstanding. I'll do it too :)

Reply:
No worries, I'm also not clear about my request :)

Reply:
I'm good with any of your preferred solution.

Reply:
Done :)